frontend/src/bundles/talent-onboarding/components/candidate-profile/profile-first-section/profile-first-section.tsx

```diff
@@ -0,0 +1,238 @@
+import mockedProjectPicture from '~/assets/img/mocked-project-picture.png';
+import {
+    Badge,
+    Button,
+    Chip,
+    Grid,
+    Typography,
+} from '~/bundles/common/components/components.js';
+import { BadgeColors } from '~/bundles/common/enums/badge-colors.enum.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+const mockedAcademyBadges = [
+    {
+        primaryText: '4.2',
+        // eslint-disable-next-line sonarjs/no-duplicate-string
```

Comment:
not a blocker, but you can add any name for mocked data. It is better than disable linter

Reply:
done